lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when', or 'with'.
+      #
+      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
+      # @see http://www.betterspecs.org/#contexts
+      #
+      # Configurable using `Prefixes` option, defaults are 'when' and 'with'.
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
```

Comment:
Maybe it can also contain an extra example showing configurable prefixes?

```yaml
Prefix:
 - with
 - without
 - when
```

```ruby
# good
context 'without display name' do
  ...
end
```

Reply:
Added an example!